src/components/Question.js

```diff
@@ -31,6 +32,8 @@ class Question extends Component {
       this.setState({
         answerSubmitted: true,
         error: false
+      }, () => {
+        currentAnswer.isCorrect ? this.setState({correctAnswer: true}) : this.setState({correctAnswer: false});
```

Comment:
forgive my nuubness, but this doesn't look right, calling setState in the setState callback. Can we not just add correctAnswer into the initial call?
```
this.setState({
  correctAnswer: currentAnswer.isCorrect,
  answerSubmitted: true,
  error: false,
});
```